deploy-helm/flux-helm-release-crd.yaml

```diff
@@ -9,5 +9,27 @@ spec:
     kind: FluxHelmRelease
     listKind: FluxHelmReleaseList
     plural: fluxhelmreleases
+    shortNames:
+    - fhr
   scope: Namespaced
   version: v1alpha2
+  versions:
+    - name: v1alpha2
+      served: true
+      storage: true
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+            - releaseName
```

Comment:
This isn't required -- it's only used if you want to override the release name that would be generated from the namespace and name.